posthog/models/team/team.py

```diff
@@ -401,13 +411,27 @@ def aggregate_users_by_distinct_id(self) -> bool:
     # DEPRECATED: use `revenue_analytics_config` property instead
     revenue_tracking_config = models.JSONField(null=True, blank=True)
 
+    # Consolidated base currency for all analytics (revenue, marketing, etc.)
+    base_currency = models.CharField(
+        max_length=3,
+        choices=CURRENCY_CODE_CHOICES,
+        default=DEFAULT_CURRENCY,
+    )
```

Comment:
You'll need `null=True` here to satisfy CI
```suggestion
    base_currency = models.CharField(
        max_length=3,
        choices=CURRENCY_CODE_CHOICES,
        default=DEFAULT_CURRENCY,
        null=True,
        blank=True,
    )
```

Reply:
Then regenerate migration